src/Arcus.WebApi.Logging/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -28,41 +28,6 @@ public static IServiceCollection AddHttpCorrelation(this IServiceCollection serv
             return AddHttpCorrelation(services, configureOptions: (HttpCorrelationInfoOptions options) => { });
         }
 
-        /// <summary>
-        /// Adds operation and transaction correlation to the application.
-        /// </summary>
-        /// <param name="services">The services collection containing the dependency injection services.</param>
-        /// <param name="configureOptions">The function to configure additional options how the correlation works.</param>
-        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="services"/> is <c>null</c>.</exception>
-        [Obsolete("Use the " + nameof(AddHttpCorrelation) + " method overload with the " + nameof(HttpCorrelationInfoOptions) + " instead")]
```

Comment:
This method has been made deprecated for a while now, I think we can indeed remove it right now.
What's your opinion @stijnmoreels  ?

Reply:
I've made a report recently about the deprecated changes on Arcus Web API:

addjwtauthorization > deprecated since 07/2022, v1.6
addcertificateauthentication > deprecated since 07/2022, v1.6
addsharedaccesskyauthentication > deprecated since 07/2022, v1.6
addhttpcorrelation options > deprecated since 03/2022, v1.5
azuremanagedidentityauthorizationfileter, deprecated since 05/2022, v1.0
sharedaccesskeyauthenticationattribute, deprecated since 01/2020, v0.4
httpcorrelation ctor, deprecated since 05/2022, v1.5

It is true that we only have two releases (on average) that show this deprecation. We did, however, already merged a PR that included a breaking change: https://github.com/arcus-azure/arcus.webapi/pull/420, so technically, we are already working on the v2.0 release.

For me, a single minor release with a deprecation notice is enough, and any new major shows breaking changes, regardless, IMHO.

So, it is fine by me that we merge these removals.

@Roeland54 , see my [comment](https://github.com/arcus-azure/arcus.webapi/pull/438#issuecomment-1453059549) in one of your other PR's: make sure that you first check with us if you want to contribute so we can assign the issue to you. Any bug/issues you added yourself, you can of course already see how it can be picked up.
Great enthusiasm, though! 🎉